src/webview-scripts/types.ts

```diff
@@ -164,3 +170,37 @@ export const enum ATNStateType {
     PLUS_LOOP_BACK = 11,
     LOOP_END = 12,
 }
+
+export enum SymbolKind {
+    Unknown,
+
+    Terminal,
+    Keyword,
+    TokenVocab,
+    Import,
+    BuiltInLexerToken,
+    VirtualLexerToken,
+    FragmentLexerToken,
+    LexerRule,
+    BuiltInMode,
+    LexerMode,
+    BuiltInChannel,
+    TokenChannel,
+    ParserRule,
+    Operator,
+    Option,
+    TokenReference,
+    RuleReference,
+    LexerCommand,
+
+    // Native code.
+    GlobalNamedAction,
+    LocalNamedAction,
+    ExceptionAction,
+    FinallyAction,
+    ParserAction,
+    LexerAction,
+    ParserPredicate,
+    LexerPredicate,
+    Arguments,
+}
```

Comment:
Why's that here? It should be possible to import that enum from src/backend/types.ts.

Reply:
I'm having trouble importing from backend.

There a comment on line 152
> // Temporary definitions for the ATN state types, because the current definition in antlr4ng requires an import of that
> // package which is not possible in the webview module.

I have assumed this is the same issue.

<img width="786" alt="image" src="https://github.com/mike-lischke/vscode-antlr4/assets/769330/f92ecee8-a959-4f33-b51a-88b863446758">
<img width="783" alt="image" src="https://github.com/mike-lischke/vscode-antlr4/assets/769330/238ab054-e513-40e4-8cd5-b8f76896ed54">

Reply:
@mike-lischke please resolve or point me in an appropriate direction.